### PR TITLE
Pages on plans and support, mostly

### PR DESCRIPTION
**Pull request by hcourdent**

### CODE REVIEW

**Comment by hcourdent** [comment]
As always, my build works locally

File: docs/misc/12_support_and_sla/index.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- P3 (Priority 3): Low-priority issues that have minimal impact on functionality or usability.
+
+## Service Level Agreement (SLA)
+
+Our SLA defines the expected response times for each priority level:
+
+- P0: We strive to respond within the same business day.
+- P1: Our goal is to respond within 24 hours.
+- P2: We target a response time of 48 hours.
+- P3: We aim to respond within 72 hours.
```
Comment (rubenfiszel) [inline docs/misc/12_support_and_sla/index.mdx:27]: 4 ways of saying the same things, extract the text above and just put time instead

you can put 3h in p0

File: docs/misc/12_support_and_sla/index.mdx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+## Security
+
+Windmill adheres to SOC 2 Type I compliance. We are currently in the process of completing our Type II audit to further enhance our security measures. As part of our commitment to maintaining a secure platform, we conduct regular penetration testing. Any identified vulnerabilities are treated with utmost priority (P0) and addressed promptly, with same-day fixes being our standard practice. We prioritize the security of our users and strive to provide a safe environment for their automation needs.
+
+## Contacting Support
+
+If you require support or have any questions, you can reach out to our support team through the following channels:
+
+- Email: contact@windmill.dev
+- Discord: [Windmill Discord Community](https://discord.com/invite/V7PM2YHsPB)
```
Comment (rubenfiszel) [inline docs/misc/12_support_and_sla/index.mdx:52]: Add dedicated slack channel on the enterprise plan

File: docs/misc/12_support_and_sla/index.mdx [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import DocCard from '@site/src/components/DocCard';
+
+# Support & SLA
+
+Windmill is committed to delivering comprehensive support and maintaining superior service levels. Our Support & Service Level Agreements (SLAs) are specifically designed to provide timely assistance and address any issues or concerns that may arise during your usage of the Windmill platform.
```
Comment (rubenfiszel) [inline docs/misc/12_support_and_sla/index.mdx:5]: sounds a bit like a word salad

File: docs/misc/12_support_and_sla/index.mdx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+## Issue Prioritization
+
+To effectively handle and prioritize issues, we categorize them into different priority levels based on their impact and urgency. This enables us to allocate resources efficiently and ensure that critical issues receive immediate attention.
+
+- P0 (Priority 0): Reserved for blocking and urgent issues that require immediate resolution.
+- P1 (Priority 1): High-priority issues that significantly impact functionality or performance.
+- P2 (Priority 2): Medium-priority issues that have a moderate impact on functionality.
+- P3 (Priority 3): Low-priority issues that have minimal impact on functionality or usability.
+
```
Comment (rubenfiszel) [inline docs/misc/12_support_and_sla/index.mdx:19]: should mention here that we track issues through private issues boards and bug trackers such as github ones.

**Pull request merged**